js/index.js

```diff
@@ -1 +1,140 @@
 // Your code goes here
+
+// Select nav tags
+
+const navTags = document.querySelectorAll('a');
+const homeNav = navTags[0];
+const aboutNav = navTags[1];
+const blogNav = navTags[2];
+const contactNav = navTags[3];
+
+// Images
+
+const images = document.querySelectorAll('.img-content')
+const adventureImg = images[0];
+const funImg = images[1];
+
+// Change background colour
+
+homeNav.addEventListener("mouseover", () => { 
+    const curColour = document.body.style.backgroundColor;
+  
+    if (curColour != 'red') {
+      document.body.style.backgroundColor = "red";
+    }
+    else {
+      document.body.style.backgroundColor = "blue";
+    }
+  });
+
+// Remove intro (display none added to mixin)
+
+const myIntro = document.querySelector('.intro p');
+aboutNav.addEventListener('dblclick', () => {
+    myIntro.classList.toggle('display-none'); 
+});
+
+// Turn intro text green when scrolling over content section
+
+const contentSection = document.querySelector('.content-section');
+contentSection.addEventListener("wheel", () => {
+    myIntro.style.color = "green";
+})
+
+// Alert message when adventure image starts being dragged
+
+adventureImg.addEventListener("dragstart", () => {
+    alert("Whooaaa - what you doing there?")
+})
+
+// When page loads, move contact nav (I've added greensock script to HTML)
+
+window.addEventListener("load", () => {
+    TweenMax.to(contactNav, 1, {
```

Comment:
Good job on implementing Greensock, this animation in particular is really clean.